### PR TITLE
add documentation for GET /users/{user_id}/status - user status 

### PR DESCRIPTION
**Pull request by cpanato**
implementation PR: https://github.com/mattermost/platform/pull/5824

### CODE REVIEW

File: v4/source/status.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      summary: Get user status by id
+      description: |
+        Get user status by id from the server.
+        ##### Permissions
+        No permission required.
+      responses:
+        '200':
+          description: User status retrieval successful
+          schema:
+            type: string
```
Comment (jwilander) [inline v4/source/status.yaml]: Can you update this to be a ref to the Status object? Might need to create it in definitions if it doesn't exist

File: v4/source/status.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+  /users/{user_id}/status:
+    get:
+      tags:
+        - status
+      summary: Get user status by id
```
Comment (jwilander) [inline v4/source/status.yaml]: Can we simplify this to just `Get user status`?

File: v4/source/status.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      description: |
+        Get user status by id from the server.
+        ##### Permissions
+        No permission required.
```
Comment (jwilander) [inline v4/source/status.yaml]: For permission can we say `Must be authenticated.`?

File: v4/source/status.yaml (hunk 2 above)
Comment (cpanato) [inline v4/source/status.yaml]: done

File: v4/source/status.yaml (hunk 3 above)
Comment (cpanato) [inline v4/source/status.yaml]: done

File: v4/source/status.yaml (hunk 1 above)
Comment (cpanato) [inline v4/source/status.yaml]: done

File: v4/source/status.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+      description: |
+        Get user status by id from the server.
+        ##### Permissions
+        Must be authenticated.
+      responses:
+        '200':
+          description: User status retrieval successful
+          schema:
+            type: '#/definitions/Status'
+        '401':
```
Comment (jwilander) [inline v4/source/status.yaml]: Sorry, one last thing, can you add a 400 BadRequest response as well?

**Pull request merged**